homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -86,27 +85,28 @@ def _get_location_from_attributes(state):
 class WazeTravelTime(Entity):
     """Representation of a Waze travel time sensor."""
 
-    def __init__(self, name, origin, destination, region,
-                 incl_filter, excl_filter, realtime):
+    def __init__(self, name, origin, destination, waze_data):
         """Initialize the Waze travel time sensor."""
         self._name = name
-        self._region = region
-        self._incl_filter = incl_filter
-        self._excl_filter = excl_filter
-        self._realtime = realtime
+        self._waze_data = waze_data
         self._state = None
         self._origin_entity_id = None
         self._destination_entity_id = None
 
-        if origin.split('.', 1)[0] in TRACKABLE_DOMAINS:
+        # Attempt to find entity_id without finding address with period.
+        pattern = "(?<![a-zA-Z0-9 ])[a-z_]+[.][a-zA-Z0-9_]+"
```

Comment:
In a previous post @balloob requested a version that didn't rely on TRACKABLE_DOMAINS, seeing that this will work as long as the entity is the following: a "lat,long" string, an entity that contains latitude and longitude attributes, or the friendly name of a zone.

Reply:
ah ok. I've been looking into making a time travel component so was interested in this PR. We have a discussion here  https://github.com/home-assistant/architecture/issues/166